.eslintrc.js

```diff
@@ -49,6 +49,13 @@ module.exports = {
   },
 
   rules: {
+    camelcase: [
+      'error',
+      {
+        properties: 'never',
```

Comment:
thanks for making this explicit!

Makes sense to me considering we do deal with a fair share of external APIs. I don't know the codebase well enough to propose enforcing this on certain files or directories